src/Utils.ps1

```diff
@@ -104,9 +125,18 @@ function Add-PoshGitToProfile {
         $TestParams
     )
 
+    if ($AllUsers -and !(Test-Administrator)) {
+        throw 'Installing posh-git for all users requires an elevated host.'
```

Comment:
What do you think about rewording that as `'Adding posh-git to an AllUsers profile requires an elevated host.'`?

Reply:
Other than that, this PR looks good to me.